docs/HyperIndex/Advanced/performance/benchmarking.md

```
@@ -18,209 +18,216 @@ Benchmarking is a critical tool for understanding and optimizing your indexer's
-pnpm envio benchmark-summary
+pnpm envio metrics          # indexing metrics from /metrics
+pnpm envio metrics runtime  # Node.js process metrics (CPU, memory, GC, event-loop lag)
 ```
 
-This command processes the collected benchmark data and displays a comprehensive performance report.
+The output is plain Prometheus text — one line per metric, with labels in braces:
 
-## Understanding Benchmark Output
-
-The benchmark output is divided into several sections, each providing insights into different aspects of your indexer's performance:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)